src/utils/configuration.ts

```diff
@@ -22,7 +29,9 @@ const configuration = (relativePath = '.percy.yml'): Configuration => {
     // in a chain. snapshot specific options -> agent configuration -> default values
 
     const defaultConfiguration: Configuration = {
-      version: 1.0, snapshot: {},
+      version: 1.0,
```

Comment:
Should this version be updated since it introduces `static_site`? It is still 100% compatible with the existing format

Reply:
I think so, 1.1?